modules/light-clients/07-celestia/update_test.go

```diff
@@ -0,0 +1,80 @@
+package celestia_test
```

Comment:
I took this test from https://github.com/cosmos/ibc-go/pull/6053 to check that the testing setup worked.